packages/base-types/src/version/settings.ts

```diff
@@ -28,16 +28,22 @@ export interface BaseVersionSettings<Prompt = unknown> {
   repeat: RepeatType;
   session: RestartSession | ResumeSession<Prompt>;
   defaultCanvasNodeVisibility: Nullable<Utils.CanvasNodeVisibility>;
+  testToolMessageDelay: number;
+  testToolMessageDelayEnabled: boolean;
```

Comment:
yes, we want these to be separated, based on designs 
https://app.zeplin.io/project/5cfebcae362911161ad0c2ca/screen/61db2941e7d9b62c3e3681fa